service/src/main/resources/schema/development/schema.json

```diff
@@ -69,8 +68,8 @@
     },
     "prov:wasGeneratedBy": {
```

Comment:
What does it mean to have `prov:wasGeneratedBy` in `required` above if it's an array type? Can it be an empty array? Do we want instead to require at least one element (if that's possible)?

Reply:
Note, require minimum size